Makefile

```diff
@@ -0,0 +1,20 @@
+
+.PHONY: help
+help:
+	@echo "make [TARGETS...]"
+	@echo
+	@echo 'Targets:'
+	@awk 'match($$0, /^([a-zA-Z_\/-]+):.*? ## (.*)$$/, m) {printf "  \033[36m%-30s\033[0m %s\n", m[1], m[2]}' $(MAKEFILE_LIST) | sort
+	@echo
+	@echo 'Internal Targets:'
+	@awk 'match($$0, /^([a-zA-Z_\/-]+):.*? ### (.*)$$/, m) {printf "  \033[36m%-30s\033[0m %s\n", m[1], m[2]}' $(MAKEFILE_LIST) | sort
+
+.PHONY: unit-tests
+unit-tests:  ## Run all tests
+	#go test -race -covermode=atomic -coverprofile=coverage.txt ./...
```

Comment:
Leftover?

Reply:
nope that's what I mentioned in the description - it should be `-race` but fails. I'm trying to fix those…

Reply:
(so fix them and enable `-race`)
but, yes, could also be remove in the meantime